maxMoviesInYear.js

```diff
@@ -0,0 +1,50 @@
+const mockData= require('./mockResult.js')
+
+function findMaxMoviesInYear () {
+    const moviesInAYear = {};
+    for ( let movie of mockData ) {
+        movie.license.reduce( ( acc, currentValue,currentIndex ) => {  
```

Comment:
Nice use of `reduce`!

Reply:
Thanks, But i actually removed reduce in my current version of code.